web/src/main/java/com/marklogic/hub/web/web/LoadDataController.java

```diff
@@ -0,0 +1,51 @@
+package com.marklogic.hub.web.web;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping("/api/current-project/loadData")
```

Comment:
@wooldridge @xnikhil08 Curious about your thoughts here. "current-project" doesn't seem meaningful to me, or rather it seems redundant. There's no concept of e.g. "mapping artifacts" that aren't part of a Data Hub application/project. So I recommend removing "current-project" from the APIs, which also seems like a nice way of defining a new set of APIs with no overlap with those used for QuickStart?

I'm also thinking "artifact" would be a useful term to include here - e.g. "/api/artifact/loadData" - thoughts on that?